pkgs/development/libraries/pipewire/wireplumber.nix

```
@@ -67,6 +67,7 @@ stdenv.mkDerivation rec {
     "-Delogind=disabled"
     "-Ddoc=${mesonEnableFeature enableDocs}"
     "-Dintrospection=${mesonEnableFeature enableGI}"
+    "-Dsystemd-system-service=true"
```

```suggestion
    "-Dsystemd-system-service=true"
    "-Dsystemd-system-unit-dir=${placeholder "out"}/lib/systemd/system"
```

Otherwise it fails to build with:

```
PermissionError: [Errno 13] Permission denied: '/nix/store/...-systemd-249.7/lib/systemd/system'
```

---

Interesting, it built fine here.

---

OK, I think I found the problem.

---

Then just disregard my suggestion 🙂

---

I've just applied it for now, but I think this is kind of hacky, and there's a better solution we can upstream.